earth2studio/data/utils.py

```diff
@@ -288,6 +288,7 @@ def datasource_to_file(
     backend: Literal["netcdf", "zarr"] = "netcdf",
     chunks: dict[str, int] = {"variable": 1},
     dtype: np.dtype | None = None,
+    **kwargs: typing.Any,
```

Comment:
Hey @gertln 

To make this a little more consistent with the IO backends, can you switch to a dictionary? E.g.
https://github.com/NVIDIA/earth2studio/blob/main/earth2studio/io/netcdf4.py#L47

Could calll it `backend_kwargs` I think.

Using an explicit dick little more explicit of a data object and keeps the options for the export separated from the function itself.